ui/app/AppLayouts/Chat/views/CreateChatView.qml

```diff
@@ -64,12 +73,16 @@ Page {
                 }
 
                 onConfirmed: {
-                    root.rootStore.createChatInitMessage = chatInput.textInput.text
-                    root.rootStore.createChatFileUrls = chatInput.fileUrls
-                    createChat()
+                    if (contactsList.visible) {
```

Comment:
This logic is wrong, see why:

https://user-images.githubusercontent.com/33099791/194327912-507dbcdc-88a1-4d4c-82f3-8e5a2d2bdef2.mp4